Sources/Networking/Core/Upload/UploadAPIManager.swift

```diff
@@ -0,0 +1,286 @@
+//
+//  UploadAPIManager.swift
+//  
+//
+//  Created by Tony Ngo on 12.06.2023.
+//
+
+import Combine
+import Foundation
+
+/// Default upload API manager
+open class UploadAPIManager: NSObject {
+    // MARK: - Public Properties
+    public var allTasks: [UploadTask] {
```

Comment:
We decided to go with activeTasks instead of allTasks for the DownloadAPIManager as well (in the docc PR for now).